bscp

```diff
@@ -121,25 +121,34 @@ def bscp(local_filename, remote_host, remote_filename, blocksize, hashname):
         (remote_size,) = struct.unpack('<Q', io.read(8))
         if remote_size < size:
             raise RuntimeError('Remote size less than local (local: %i, remote: %i)' % (size, remote_size))
-        remote_digest_list = [io.read(hash_total.digestsize) for i in xrange(blockcount)]
-
-        for remote_digest in remote_digest_list:
+        remote_digest_list = [(i,io.read(hash_total.digestsize),
+            sys.stderr.write((''*42)+'Read remote digest [%10d/%10d]'%(i,blockcount)),sys.stderr.flush()
+            ) for i in xrange(blockcount)]
+        sys.stderr.write('\n')
+        for block_num,remote_digest,write,flush in remote_digest_list:
+            sys.stderr.write((''*42)+'Digest local block [%10d/%10d]'%(block_num,blockcount))
+            sys.stderr.flush()
```

Comment:
This is a bit cumbersome. Also, flush() is unnecessarily called twice. Please simplify this code, e.g. like this:

```python
for i in xrange(blockcount):
    remote_digest = io.read(hash_total.digestsize)
    sys.stderr.write(...)
    sys.stderr.write(...)
    sys.stderr.flush()
    ...